sbws/util/fs.py

```diff
@@ -0,0 +1,50 @@
+"""Utils file system functions"""
+
+import logging
+import shutil
+
+log = logging.getLogger(__name__)
+
+
+def sbws_required_disk_space(conf):
+    """Disk space required by sbws files.
+    Rough calculations.
+
+    :param ConfigParser conf: sbws configuration
+    :returns: int, size in MiB
+    """
+    # Number of relays per line average size in Bytes
+    size_v3bw_file = 7500 * 220
+    num_v3bw_files = int(conf['general']['data_period'])
```

Comment:
data_period is the number of days into the past that we consider measurements valid. Why does this line assume that the number of v3bw files will be the same as this value?

Reply:
I think that i intended to calculate size of the minimum number of results files to keep, not v3bw files.
For v3bw files, it would be enough to keep only the last 2 ones that should have been generated in the last 2 hours. 
This is going to depend a lot on the crontab that will generate/clean files and would be provided separately, but at least this would give an estimation on the minimum space required